IndicoToolkit/IndicoWrapper/Workflows.cs

```diff
@@ -1,13 +1,201 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using System.IO;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
 using IndicoV2;
+using IndicoV2.Workflows.Models;
+using IndicoV2.Submissions.Models;
 
-namespace IndicoToolkit
+using IndicoToolkit.Types;
+using IndicoToolkit.Exception;
+
+namespace IndicoToolkit.IndicoWrapper
 {
-    public class Workflows
+    /// <summary>
+    /// Class <c>Workflows</c> supports Workflow-related API calls
+    /// </summary>
+    public class Workflows : IndicoClientWrapper
     {
-        public IndicoClient client;
-        public Workflows(IndicoClient _client)
+        public Workflows(IndicoClient _client) : base(_client) { }
+
+
+        /// <summary>
+        /// Retrieves workflow with given datasetID
+        /// </summary>
+        /// <param name="datasetId">ID of corresponding dataset</param>
+        public async Task<IEnumerable<IWorkflow>> GetWorkflow(int datasetId)
```

Comment:
do we need this call? just thinking that this call might be antiquated... i.e. multiple workflows can be associated with one datasetId

Reply:
makes sense